tools/telemetry-generator/.eslintrc.cjs

```diff
@@ -21,4 +21,16 @@ module.exports = {
 		// This package is exclusively used in a Node.js context
 		"import/no-nodejs-modules": "off",
 	},
+	overrides: [
+		{
+			// Rules only for test files
+			files: ["*.spec.ts", "src/test/**"],
+			rules: {
+				// Disabling for now
+				"@typescript-eslint/no-require-imports": "off",
+				"@typescript-eslint/no-var-requires": "off",
+				"import/no-internal-modules": "off",
+			},
+		},
+	],
```

Comment:
All of these things complain today in main. Not sure if we can make the handlers ESM, so until we investigate that, just disabling the rules.

Reply:
Can we file a work item and reference it in a TODO comment?

Reply:
Done.